vms/avm/unique_tx.go

```diff
@@ -324,7 +324,7 @@ func (tx *UniqueTx) SemanticVerify() error {
 
 	return tx.Unsigned.Visit(&executor.SemanticVerifier{
 		Backend: tx.vm.txBackend,
-		State:   tx.vm.dagState,
+		State:   tx.vm.state,
```

Comment:
By removing the usage of `dagState` here - we will no longer allow tx verification to occur based on `Processing` state. This is fine - as we only ever verify transactions like this during bootstrapping now - which is performed in topological order with `Verify` immediately followed by `Accept`.